go/billing/api.py

```diff
@@ -406,7 +406,7 @@ def check_and_notify_low_credit_threshold(
         if level is not None:
             return spawn_celery_task_via_thread(
                 create_low_credit_notification, account_number,
-                level, credit_balance)
+                level, credit_balance, level*100 == self._notification_mapping[0])
```

Comment:
I think it would be good to do ``cutoff_notification = (level * 100 == self._notification_mapping[0])`` just to make it clear what the value being passed in is for.